### ISSUE TITLE
Multiple stores have been displaced since 1.54

### ISSUE THREAD

**Issue by Gopher-nz** [A3Wasteland/ArmA3_Wasteland.Altis#363]
These include: 
Vehicle stores - north east airfield, and west coast
Gun stores - pyrgos
General stores - molos, 

Those are the ones I am aware of.Any suggestions on how to fix?

**Comment by thegoatman** [comment]
Yes relocated them with the editor or wait untill BIS releases a new patch to fix the buildings

**Comment by Gopher-nz** [comment]
Hmm new patch made no difference. Would a happily relocate the stores but believe it or not struggle to get my mission PBO using into the editor.

**Comment by thegoatman** [comment]
The patch released today was only for an battleye update ( http://steamcommunity.com/games/arma3/announcements/detail/101692901110787310 )

and check the A3W forums there are plenty of tutorials there how to place / change stores

**Comment by Gopher-nz** [comment]
I have done already but always have a problem getting my pbo into the editor. Besides given this issue was triggered by an update - what are the chances of a fix undoing all the changes I make. I am currently in the "wait for a fix" camp unless BI absolve themselves of all responsibility and then I will go down the dark road of editing afterwards.

**Comment by dewitjur** [comment]
i got it in the editor by making a pbo with only the mission.sqf in it, afterwards i placed it back in the misison file.

**Comment by Gopher-nz** [comment]
Thanks demitjur - I did too after your suggestion. And do you know the bizarre thing? Ever store owner is in their correct place in the editor. I load up the mission - and store owners mentioned before are displaced. wtf.

**Comment by hobart666** [comment]
That is because it is an issue within the game engine more than likely and
is something that Bohemia has to correct. This same thing has happened
before after one of their "updates" and we had to wait for them to fix it.

**Comment by thegoatman** [comment]
Think this issue can be closed its fixed in the latest RC build. Just tested and confirmed it

https://forums.bistudio.com/topic/183855-release-candidate-branch-announcements/?p=2949173

**Comment by Gopher-nz** [comment]
Cant wait for it to be released. Any ideas when that will be.

**Comment by hobart666** [comment]
This seems to be fixed at least on my server.

**Comment by dewitjur** [comment]
On my server, it keeps changing the location for the symbols to wrong and right but store itseld is always misplaced. I once tried without the building protection script and it messed up more
